centreon/tests/e2e/features/Resources-Access-Management/commands.ts

```diff
@@ -0,0 +1,36 @@
+Cypress.Commands.add('createMultipleResourceAccessRules', (numberOfTimes) => {
+  for (let i = 1; i <= numberOfTimes; i++) {
+    const name = `Rule${i}`;
+    const payload = {
+      contact_groups: { all: false, ids: [] },
+      contacts: { all: false, ids: [17] },
+      dataset_filters: [
+        { dataset_filter: null, resources: [14], type: 'host' }
+      ],
+      description: '',
+      is_enabled: true,
+      name: name
+    };
+    cy.request({
+      body: payload,
+      method: 'POST',
+      url: '/centreon/api/v24.04/administration/resource-access/rules?*'
```

Comment:
It's not better to render a variable of api version?

Reply:
yes you are right